packages/block-library/src/query-pagination-next/edit.js

```diff
@@ -17,28 +17,29 @@ export default function QueryPaginationNextEdit( {
 } ) {
 	const displayArrow = arrowMap[ paginationArrow ];
 	return (
-		<a
-			href="#pagination-next-pseudo-link"
-			onClick={ ( event ) => event.preventDefault() }
-			{ ...useBlockProps() }
-		>
-			<PlainText
-				__experimentalVersion={ 2 }
-				tagName="span"
-				aria-label={ __( 'Next page link' ) }
-				placeholder={ __( 'Next Page' ) }
-				value={ label }
-				onChange={ ( newLabel ) =>
-					setAttributes( { label: newLabel } )
-				}
-			/>
-			{ displayArrow && (
-				<span
-					className={ `wp-block-query-pagination-next-arrow is-arrow-${ paginationArrow }` }
-				>
-					{ displayArrow }
-				</span>
-			) }
-		</a>
+		<div { ...useBlockProps() }>
```

Comment:
This change looks unrelated. What’s the issue here? We should rather process it independently.

Reply:
If we don't put a parent div there, the option of changing link color on the editor won't work. It is not working now on Query Pagination Next and Previous links. 😔

With a parent `div`, a link works.
<img width="1181" alt="Screenshot 2021-11-29 at 11 23 19" src="https://user-images.githubusercontent.com/37012961/143851093-2e77ca64-9ad8-450a-8bff-5ef3ff6c4f4f.png">

Without, it won't:
<img width="1211" alt="Screenshot 2021-11-29 at 11 24 12" src="https://user-images.githubusercontent.com/37012961/143851238-ad50094b-f1d0-4c5b-a314-f6c0a72be5e0.png">

Reply:
@ntsekouras, is it something on the radar? It looks like a bug fix worth considering for WP 5.9.

Reply:
Good catch! I opened a PR proposing to remove this support from these blocks here: https://github.com/WordPress/gutenberg/pull/36954